content/verification/veteran-verification/release-notes/2024-05-13.md

```diff
@@ -0,0 +1,5 @@
+We updated the results returned for disability ratings. They will now exclude ratings that do not have a “begin” date.
+Added fields  (underline)
```

Comment:
What does "Added fields (underline)" mean? I need more clarity here.

Reply:
Got some clarification from Lorri. She was just referencing that we added the new field **hyph_diagnostic_type_code** and she meant underscore. We can remove the line:
Added fields (underline)